fileattrs/sysusers.attr

```diff
@@ -20,3 +20,41 @@
         ::continue::
     end
 }
+
+%__sysusers_recommends() %{lua:
+    if rpm.expand('%[0%{?_use_weak_usergroup_deps}]') == '0' then
+       return
+    end
+    for line in io.lines(macros["1"]) do
+        if line:sub(1, 1) == '#' then
+            goto continue
+        end
+        fields = {}
+        for w in line:gmatch("%S+") do
+            table.insert(fields, w)
+        end
+        if #fields >= 3 and fields[1] == 'm' then
+	    print(string.format('user(%s)\\ngroup(%s)', fields[2], fields[3]))
+        end
+        ::continue::
+    end
+}
+
+%__sysusers_requires() %{lua:
+    if rpm.expand('%[0%{?_use_weak_usergroup_deps}]') ~= '0' then
+       return
+    end
+    for line in io.lines(macros["1"]) do
```

Comment:
Parametric macro arguments are available as `arg[1]` etc, no need to go through `macros`.